src/code/irqmgr.c

```diff
@@ -127,54 +179,64 @@ void IrqMgr_CheckStack() {
 }
 
 void IrqMgr_HandlePRENMI450(IrqMgr* this) {
-    u64 temp = STATUS_NMI; // required to match
-    gIrqMgrResetStatus = temp;
-    this->resetStatus = STATUS_NMI;
+    u64 nmi = IRQ_RESET_NMI; // required to match
+
+    gIrqMgrResetStatus = nmi;
+    this->resetStatus = IRQ_RESET_NMI;
```

Comment:
This can't be a chained assignment, right?

Reply:
Tried it, doesn't seem to work